ui/app/src/components/SearchBar/SearchBar.tsx

```diff
@@ -122,7 +122,7 @@ export function SearchBar(props: SearchBarProps) {
   const [focus, setFocus] = useState(false);
   const { formatMessage } = useIntl();
   const finalPlaceholder = placeholder || formatMessage(messages.placeholder);
-  const inputRef = useRef<HTMLInputElement>(undefined);
+  const inputRef = useRef<HTMLInputElement | null>(undefined);
```

Comment:
There's no type consistency here. Either the type should be `Element | undefined` or the ref should be initialised to `null` instead of `undefined`.
```suggestion
  const inputRef = useRef<HTMLInputElement | null>(null);
```